config/hubs/pangeo-hubs.cluster.yaml

```diff
@@ -74,6 +72,15 @@ hubs:
                   - choldgraf
                   - rabernat
                 admin_users: *staging_users
+              JupyterHub:
+                authenticator_class: github
+              GitHubOAuthenticator:
+                oauth_callback_url: https://staging.pangeo.2i2c.cloud/hub/oauth_callback
+                allowed_organizations:
+                  - pangeo-data:us-central1-b-gcp
+                  - 2i2c-org:tech-team
```

Comment:
Since the tech team is here now, maybe we can remove the allowed_users from above and just keep them as admin_users? Am I correct in my interpretation?

Reply:
I don't think so no. I don't think `allowed_organizations` grants admin privileges. It would be a nice to have though!

Reply:
Sorry, just re-read this and parsed it properly. I _thought_ you meant removing the admin block! Yes, we can probably remove the `allowed_users` block.

Reply:
You are in fact correct and this block needs to go, otherwise it prevents folk from logging in _even if_ they are a member of the appropriate org/team. Fixed in https://github.com/2i2c-org/pilot-hubs/pull/707/commits/ee39b165fa5a91c61517ff2ad9d441e0d446ddef

Reply:
Nice to know my intuition is still accurate 😜  .